types/address_test.go

```diff
@@ -519,3 +570,385 @@ func (s *addressTestSuite) TestMustAccAddressFromBech32() {
 
 	s.Require().Equal(valAddress1, valAddress2)
 }
+
+func (s *addressTestSuite) TestMustValAddressFromBech32Panic() {
+	s.Require().Panics(func() {
+		types.MustValAddressFromBech32("")
+	})
+}
+
+func (s *addressTestSuite) TestGetBech32PrefixAccPub() {
+	actual := types.GetBech32PrefixAccPub("")
+	s.Require().Equal(types.PrefixPublic, actual)
+
+	actual = types.GetBech32PrefixAccPub("cosmos")
+	expected := "cosmos" + types.PrefixPublic
+	s.Require().Equal(expected, actual)
+}
```

Comment:
Thank you. Now I understand better. 

I decided to use type.PrefixPublic in the test because I though that it is better to test the behaviour of the function not concrete state. What I mean? If someone change the value of the types.PrefixPublic the test will not fail because it tests the behaviour of the function not concrete state.

Now I realise that  types.PrefixPublic is supposed to be stable and its exact value is critical for the correctness of the application, state testing with hardcoded values is more appropriate. This approach ensures that any unintended changes are caught immediately. 

All the changes are applied.